plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx

```diff
@@ -57,6 +57,19 @@ const generatedColumns: TableColumn[] = [
       return <>{row.groupOffset ?? ''}</>;
     },
   },
+  {
+    title: 'Lag',
+    field: 'lag',
+    render: (row: Partial<TopicPartitionInfo>) => {
+      let lag = undefined;
+      if (row.topicOffset && row.groupOffset) {
+        const topicOffset = +row.topicOffset;
+        const groupOffset = +row.groupOffset;
+        lag = topicOffset - groupOffset;
```

Comment:
i'll admit to not being familiar with this particular formatting of the plus prefix (is this just to force an addition?

but since those two vars are locally scoped to the if statement, this could presumably be more easily written like:

```js
lag = +row.topicOffset - +row.groupOffset;
```

or just

```js
lag = row.topicOffset - row.groupOffset;
```

No?  again i may have missed some context so apols

Reply:
hi @adamdmharvey the `+` sign just converts from string to number in order to be able to do the math. I think i can condense it to a one line as you suggest. 
Like:
``` 
lag = +row.topicOffset - +row.groupOffset;
```

Reply:
Could be a bit clearer to use the Number constructor perhaps, but it may be a bit of a matter of taste.

```ts
Number(row.topicOffset)
```